tests/unit/aiplatform/test_training_jobs.py

```diff
@@ -1397,6 +1502,47 @@ def test_run_call_pipeline_service_create_with_enable_web_access(
             gca_pipeline_state.PipelineState.PIPELINE_STATE_SUCCEEDED
         )
 
+    @pytest.mark.usefixtures(
+        "mock_pipeline_service_create_with_scheduling",
+        "mock_pipeline_service_get_with_scheduling",
+        "mock_get_backing_custom_job_with_scheduling",
+        "mock_python_package_to_gcs",
+    )
+    @pytest.mark.parametrize("sync", [True, False])
+    def test_run_call_pipeline_service_create_with_scheduling(self, sync, caplog):
+
+        caplog.set_level(logging.INFO)
```

Comment:
no need caplog for testing, this applied to all 3 tests

Reply:
I see this in multiple tests though.

Reply:
Done